### ISSUE TITLE
5e 3.0 support

### ISSUE THREAD

**Issue by emckee2006** [crash1115/5e-training#100]
I have a tiny set of changes (like 3 lines) so it will work on the new 5e release.  Would you like the contribution?

**Comment by crash1115** [comment]
Hey, yeah, if you wanna open a PR I'd be glad to take a look. You might also wanna submit one over [here](https://github.com/p4535992/foundryvtt-downtime-dnd5e) as well; it's the spiritual successor to this mod, and is probably more actively maintained than this one is.

**Comment by crash1115** [comment]
FYI, I am working on this right now. It's gonna be a little more involved than I expected because they left support in place for the legacy sheet; I'm probably gonna need a new template for the content on the new 3.0 sheet, and that's an excuse to clean up the sorry state of the CSS, and that's an excuse to... My to-do list keeps snowballing lol.

TLDR don't worry about that PR. I'll try to have this out by the end of the week.

**Comment by crash1115** [comment]
Sorry for the delay, had some stuff come up.

Version 1.1.0 contains dnd5e 3.x support. You'll need to be running the latest patch (dnd5e v3.0.3 as of writing) for the module to work.
